.github/workflows/conformance-aks-v1.12.yaml

```diff
@@ -196,12 +197,10 @@ jobs:
           target_url: ${{ env.check_url }}
 
       - name: Install Cilium CLI
-        run: |
-          curl -sSL --remote-name-all https://github.com/cilium/cilium-cli/releases/download/${{ env.cilium_cli_version }}/cilium-linux-amd64.tar.gz{,.sha256sum}
-          sha256sum --check cilium-linux-amd64.tar.gz.sha256sum
-          sudo tar xzvfC cilium-linux-amd64.tar.gz /usr/local/bin
-          rm cilium-linux-amd64.tar.gz{,.sha256sum}
-          cilium version
+        uses: cilium/cilium-cli@v0.14.1
```

Comment:
I think this needs to be the SHA of the v0.14.1 tag so that renovate bot can update this dependency automatically.

Reply:
Is this nonblocking because we actually don't care and it's up to the opinion of the contributor, or it's nonblocking because we think it must be done and it just doesn't need to be done in this PR? I was under the impression that we exclusively use SHAs for supply chain security reasons, so we should just fix this up now before merge...?

Reply:
My reasoning is that we control `cilium/cilium-cli`, so SHA is not needed. Also, if we use SHA, then wouldn't any merge to cilium-cli would cause the action update PR?

Reply:
> Also, if we use SHA, then wouldn't any merge to cilium-cli would cause the action update PR?

Nope, only releases would cause renovatebot to bump the action. It would still use the SHA though. So I think there is no harm in using the release SHA.

Reply:
> > Also, if we use SHA, then wouldn't any merge to cilium-cli would cause the action update PR?
> 
> Nope, only releases would cause renovatebot to bump the action. It would still use the SHA though. So I think there is no harm in using the release SHA.

Sorry, I forgot to mention that in order for renovate to only update digests on releases a comment with the version is needed, see https://docs.renovatebot.com/modules/manager/github-actions/

I've opened #25369 to add those.